metricbeat/module/apache/status/data.go

```diff
@@ -211,40 +211,50 @@ func eventMapping(body io.ReadCloser, hostname string, metricset string) common.
 	}
 
 	event := common.MapStr{
-		"totalAccesses":           totalAccesses,
-		"totalKBytes":             totalKBytes,
-		"uptime":                  uptime,
-		"cpuLoad":                 cpuLoad,
-		"cpuUser":                 cpuUser,
-		"cpuSystem":               cpuSystem,
-		"cpuChildrenUser":         cpuChildrenUser,
-		"cpuChildrenSystem":       cpuChildrenSystem,
-		"reqPerSec":               reqPerSec,
-		"bytesPerSec":             bytesPerSec,
-		"bytesPerReq":             bytesPerReq,
-		"busyWorkers":             busyWorkers,
-		"idleWorkers":             idleWorkers,
-		"connsTotal":              connsTotal,
-		"connsAsyncWriting":       connsAsyncWriting,
-		"connsAsyncKeepAlive":     connsAsyncKeepAlive,
-		"connsAsyncClosing":       connsAsyncClosing,
-		"serverUptimeSeconds":     serverUptimeSeconds,
-		"load1":                   load1,
-		"load5":                   load5,
-		"load15":                  load15,
-		"scbStartingUp":           totalS,
-		"scbReadingRequest":       totalR,
-		"scbSendingReply":         totalW,
-		"scbKeepalive":            totalK,
-		"scbDnsLookup":            totalD,
-		"scbClosingConnection":    totalC,
-		"scbLogging":              totalL,
-		"scbGracefullyFinishing":  totalG,
-		"scbIdleCleanup":          totalI,
-		"scbOpenSlot":             totalDot,
-		"scbWaitingForConnection": totalUnderscore,
-		"scbTotal":                totalAll,
-		"hostname":                hostname,
+		"hostname":      hostname,
+		"totalAccesses": totalAccesses,
+		"totalKBytes":   totalKBytes,
+		"reqPerSec":     reqPerSec,
+		"bytesPerSec":   bytesPerSec,
+		"bytesPerReq":   bytesPerReq,
+		"busyWorkers":   busyWorkers,
+		"idleWorkers":   idleWorkers,
+		"uptime": common.MapStr{
+			"serverUptimeSeconds": serverUptimeSeconds,
```

Comment:
Request URL for both versions is always the same. It depends on configuration of the webserver and there is no flag in output page.
The difference is only list (content) of fields as output. And only this would serve in case of implementation of automatic detection.

It would be possible to group fields in two main groups "standard" and "extended". It can be also good point for metricsets. But ... if you configure it in metricbeat and don't allow on webserver, it will not work as data for extended metricset will not be available.

Then, related question is how to change nested groups. They could be divided to:
- standard
 * workers
* connections
* ...
- extended
  * load
  * cpu 
  * etc